services/payments/live_solana_test.go

```diff
@@ -83,43 +90,43 @@ func TestLiveSolanaStateMachineATAMissing(t *testing.T) {
 
 	ctx = context.WithValue(ctx, ctxAuthKey{}, "some authorization from CLI")
 
-	// Should transition transaction into the Authorized state
+	// State transition: Prepared -> Authorized
 	testState.Status = paymentLib.Prepared
 	marshaledData, _ = json.Marshal(testState)
 	mockTransitionHistory.Data.UnsafePaymentState = marshaledData
 	solanaStateMachine.setTransaction(&testState)
-	newTransaction, err := Drive(ctx, &solanaStateMachine)
+	authorizedTransaction, err := Drive(ctx, &solanaStateMachine)
```

Comment:
Suggestion: here and everywhere else, consider using less verbose language. `authTxn` conveys the same meaning, and has a higher signal/noise ratio.